Directory.Build.props

```diff
@@ -20,7 +20,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="CommunityToolkit.Maui.UnitTests" />
   </ItemGroup>
-  <PropertyGroup Condition="'$(TF_BUILD)' == 'true' and $([MSBuild]::IsOSPlatform('windows')) == 'true'">
```

Comment:
I would suggest removing this condition, otherwise, the developer will have a failed build on CI but a green build locally, which may be confusing.
as we only need to install the script once, I don't think it is an issue.

Reply:
That is weird. Is the CI build failed with including the condition? I expected Tizen to be included in the CI build and excluded on the local build.

Reply:
Looks like the build will be ok with the condition in both CI and local.

Reply:
If you miss Tizen file on local env, CI will fail.

Reply:
Oh, you meant when you add a new features? All right, I'm Ok if other members are not skeptical. 😂